checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Checklist for the victims of a data breach"
+description: We collected a comprehensive list of notes especially for the victims of the Vastaamo data breach. If you have been a client of Vastaamo before the year 2018, your confidential information may have been stolen. According to Vastaamo, copies and accessed customer records might exist from early 2019, likely until March 2019.
+---
```

Comment:
Should include an "aliases" section according to the website syntax?

Reply:
Possibly, I'm not certain.